specs/graphite/gt_branch_new_with_commit_message.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Graphite - Create a branch with an accompanying commit message
+command: "gt bc -am {{commit_message}}"
+tags:
+  - graphite
+description: "Creating some changes, staging those changes, creating a new branch and committing those changes to the new branch is way too much work. Create your changes and run this single-line command to do all of that in one go. Compare to: `git checkout -b <branch_name> && git add . && git commit -m <commit message>`"
```

Comment:
Let's simplify this to be more direct. "Create a new branch, stage all changes, and use given text as commit message, similar to ..."